File: public/codesamples/posts/graphes_connections/Node.cs
namespace Posts.GrahpesConnections
{
    public class Node
    {
        private int _x;
        private int x
        {
            get { return _x; }
        }
        private int _y;
        private int y
        {
            get { return _y; }
        }
        private HashSet<Connection> _connections = new HashSet<Connection>();
        public HashSet<Connection> connections
        {
            get { return _connections; }
        }
        public Node(int x, int y)
        {
            _x = x;
            _y = y;
        }
    }
}

File: public/codesamples/posts/graphes_connections/Node.java
package posts.graphes_connections;

import java.util.Set;
import java.util.HashSet;

public class Node {
  private int x;
  private int y;
  private Set<Connection> connections = new HashSet<>();

  public Node(int x, int y) {
    this.x = x;
    this.y = y;
  }

  public int getX() {
    return x;
  }

  public int getY() {
    return y;
  }

  public Set<Connection> getConnections() {
    return connections;
  }
}
